app/Http/Controllers/User/HomeController.php

```diff
@@ -35,6 +36,9 @@ public function index(Request $request)
         $subjects = Subject::all();
         $teachers = $teachers->paginate(10);
 
-        return view('home', compact('grades','subjects','teachers','request'));
+        return view(
+            'home',
+            compact('grades','subjects','teachers','request')
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/HomeController.php</br>**Line**: 41</br>No space found after comma in function call